From 9338f1709b3b499508717cdaa3435ff58a077b5a Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Wed, 7 May 2014 13:46:46 +0800
Subject: [PATCH 1/2] pyanaconda/bootloader.py: fix for the WR efi

Our efi is EFI/BOOT/*.efi, not EFI/redhat/grub.efi.

Upstream-Status: Inappropriate [WR Specific]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 pyanaconda/bootloader.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyanaconda/bootloader.py b/pyanaconda/bootloader.py
index 7849469..bba44f0 100644
--- a/pyanaconda/bootloader.py
+++ b/pyanaconda/bootloader.py
@@ -1361,7 +1361,7 @@ class GRUB(BootLoader):
 class EFIGRUB(GRUB):
     packages = ["grub-efi", "efibootmgr"]
     can_dual_boot = False
-    _config_dir = "efi/EFI/redhat"
+    _config_dir = "efi/EFI/BOOT"
 
     stage2_is_valid_stage1 = False
     stage2_bootable = False
@@ -1435,7 +1435,7 @@ class EFIGRUB(GRUB):
 
         rc = self.efibootmgr("-c", "-w", "-L", productName,
                              "-d", boot_disk.path, "-p", boot_part_num,
-                             "-l", "\\EFI\\redhat\\grub.efi",
+                             "-l", "\\EFI\\BOOT\\boot*.efi",
                              root=install_root,
                              stdout="/dev/tty5", stderr="/dev/tty5")
         if rc:
-- 
1.8.3.1

